third_party/bigframes_vendored/version.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 Google LLC
```

Comment:
nit: maybe 2024 ?

Reply:
Since it's copied from a filed that was copyright 2022, I'm inclined to leave this as-is. Not that there's much to copyright :-)